src/main/java/it/pagopa/selfcare/pagopa/backoffice/service/InstitutionsService.java

```diff
@@ -35,6 +36,11 @@ public InstitutionUploadData getInstitutionData(String institutionsData) {
             return institutionClient.getInstitutionData(institutionsData);
         } catch (AppException e) {
             throw e;
+        } catch (FeignException e) {
```

Comment:
not needed, we have a handler

Reply:
The returned value from the handler is not useful for this case.